### ISSUE TITLE
Improve err message for UnusedPrivateMember distinguish between property/param etc.

### ISSUE THREAD

**Issue by robstoll** [arturbosch/detekt#1243]
## Expected Behavior
The error message tells clearly what is unsed instead of using `Private property ... is unused.` all the time. For instance:
```
fun foo(unused: Int){}
```
Should report `Parameter ... unused`

```
fun foo(){ val unused = 1 }
```
Should report `Local variable ... unused` 

etc.
## Current Behavior
Uses `Private property ... is unused.` all the time. 

## Steps to Reproduce (for bugs)
See above

## Your Environment
* Version used: 1.0.0.RC8

**Comment by Mauin** [comment]
As #1272 shows this already seems to be the case.

See the calls to report in the Rule itself:

https://github.com/arturbosch/detekt/blame/master/detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedPrivateMember.kt#L68

https://github.com/arturbosch/detekt/blame/master/detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedPrivateMember.kt#L142

https://github.com/arturbosch/detekt/blame/master/detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedPrivateMember.kt#L145

**Comment by robstoll** [comment]
Cool, as stated I am using 1.0.0.RC8 if it is fixed in a newer version the better ☺. IMO this can be closed then
